example/relational/README.md

```diff
@@ -260,6 +269,10 @@ inverse := Keyword{HashKey: article, SortKey: keyword, /* ... */}
 db.Put(forward)
 db.Put(inverse)
 ```
+=> would `forward` and `inverse` have both the content of the article or only one?
+==> if only one, how the access pattern would look like? would it be a 2 reads? eg:
+1. read inverse, get `forward` partition key
+2. read forward get article content
```

Comment:
yes, 2 reads but
1. The first read uses `keyword:theory` partition key, would discover related article `article:neumann/theory_of_automata`.
2. the second read using `article:neumann/theory_of_automata` key would read `( author:neumann, article:theory_of_automata )` object